Manual.md

```diff
@@ -132,6 +135,83 @@ the Void packages system.
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
+<a id="cross_compilation"></a>
+### Cross compilation
+
+Void Linux places high emphasis on having packages being able to be
+cross-compiled.
+
+In the context of Void Linux cross compilation is compiling software from one
+CPU architecture to another, e.g. compile to aarch64 (64bits ARM) from x86_64
+(Intel/AMD 64bits) system.
```

Comment:
IMO `e.g. compiling from a x86_64 system (Intel/AMD 64-bit) to aarch64 (ARMv8 64-bit)` sounds better.